app/src/main/java/com/hyperether/getgoing/activity/ShowLocationActivity.java

```diff
@@ -908,4 +926,32 @@ public void openGPSSettings() {
         Intent i = new Intent(ACTION_LOCATION_SOURCE_SETTINGS);
         startActivityForResult(i, Constants.REQUEST_GPS_SETTINGS);
     }
+
+    private void backgroundTrackNotification(int notificationID){
+        NotificationCompat.Builder builder = new NotificationCompat.Builder(this);
+        builder.setSmallIcon(R.drawable.ic_launher);
+        builder.setContentTitle("GetGoing notification");
+        builder.setContentText("Location tracking in background");
```

Comment:
Text should not be hardcoded. Please define string in xml that can be localized.